src/map/lua/luautils.cpp

```diff
@@ -230,6 +231,13 @@ namespace luautils
         lua.set_function("RoeParseRecords", &roeutils::ParseRecords);
         lua.set_function("RoeParseTimed", &roeutils::ParseTimedSchedule);
 
+        // Fishing Contest Functions
+        lua.set_function("GetFishingContest", &luautils::GetFishingContest);
+        lua.set_function("InitNewFishingContest", &luautils::InitNewFishingContest);
+        lua.set_function("SetContestParameters", &luautils::SetContestParameters);
+        lua.set_function("ProgressFishingContest", &luautils::ProgressFishingContest);
+        lua.set_function("InitializeFishingContestSystem", &luautils::InitializeFishingContestSystem);
```

Comment:
I _think_ you'll have to add these to the Lua ci scripts if you haven't already, so it doesn't complain about unknown global funcs

Reply:
Is that `global_objects` in lua.sh?